src/store.ts

```diff
@@ -102,6 +102,14 @@ export type PixelConfiguration = Dictionary;
 export type MigrationData = Dictionary;
 export type ServerSettings = Dictionary;
 
+type WrapperSDKTypes = 'flutter' | 'none';
+interface WrapperSDKInfo {
```

Comment:
I think `WrapperSDKOptions` makes more sense

Reply:
Sure, I'll make the change

Reply:
@einsteinx2 I meant the interface, not `name`